lib/mli_parser.ml

```diff
@@ -46,62 +46,28 @@ let slice lines ~(start : Odoc_model.Location_.point)
       let last_line = String.sub last_line 0 end_.column in
       String.concat "\n" ([ first_line ] @ stripped @ [ last_line ])
 
-(* Imagine a docstring that is within a file with four characters # of indentation. (I'll
-   use square brackets rather than parens to avoid escaping):
-
-   ####[** foo
-   ####
-   ####bar
-   ####
-   ####baz *]
-   ####val x : int
-   ####val y : int
-
-   According to odoc, the "b" in "bar" is at column 0 inside the docstring and at column 4
-   within the broader file. That is correct. But it says the "f" in "foo" is at column 1
-   inside the docstring and column 5 within the file. This isn't right.
-
-   The problem is that it starts counting the inside-the-docstring column number from the
-   end of "[**", but doesn't add those three characters to the within-the-file column
-   number. Here, we make the adjustment.
-*)
-let account_for_docstring_open_token (location : Odoc_model.Location_.span) =
```

Comment:
The parser now keeps the location :)

Reply:
I talked too fast. This is still needed, an other force-push.

Reply:
I debugged this a bit more and found that there is no bug into odoc-parser but rather that it expects the starting location to account for the comment opening. That's how odoc does it: https://github.com/ocaml/odoc/blob/master/src/loader/doc_attr.ml#L56
I pushed a fix.

Reply:
Indeed, as described here: https://ocaml-doc.github.io/odoc-parser/Odoc_parser/index.html#val-parse_comment